Podfile.lock

```diff
@@ -123,4 +123,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 69fe77633cdb363bbf101efe31c9b95b5679465d
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
Downgrade ;(

Reply:
It depends on your cocoa pods version. Doesn’t affect project. Will update version later

Reply:
I think you can upgrade your local version for consistency :)

Reply:
Updated to 1.9.3